R/timeseriessummary-data.R

```diff
@@ -155,7 +155,7 @@ parseTSSRatingCurves <- function(reportData, timezone){
   
   if(!isEmptyOrBlank(curves)){
     colnames(curves)[which(colnames(curves) == 'remarks')] <- "curveRemarks"
-    
+    curves <- curves[order(curves[['startOfPeriod']]),]
```

Comment:
In the first test for this, the class of `curves[['startOfPeriod']]` is character. So using `order` could be a problem. It appears to do it correctly, but makes me nervous that it doesn't throw a warning or something. 

`order(c("2010-10-01", "2011-10-01", "2010-10-02"))`